src/components/Nav.tsx

```diff
@@ -18,7 +18,11 @@ import {
 import React, { useEffect } from "react";
 import { gql, useMutation } from "@apollo/client";
 import { HamburgerIcon } from "@chakra-ui/icons";
-import { Link as ReactRouterLink, useNavigate } from "react-router-dom";
+import {
+  Link as ReactRouterLink,
+  Link as Link2,
```

Comment:
What's going on with this lol, you should be able to just replace all usage of `Link2` with `ReactRouterLink` and then remove the `Link2` alias

Reply:
Haha I don't know what I was thinking. Will change this